libs/features/components/cards/src/lib/cards/elewa-vertical-icon-and-text/elewa-vertical-icon-and-text.component.ts

```diff
@@ -1,12 +1,27 @@
-import { Component, Input } from '@angular/core';
+import { Component } from '@angular/core';
 
 @Component({
   selector: 'elewa-group-elewa-vertical-icon-and-text',
   templateUrl: './elewa-vertical-icon-and-text.component.html',
   styleUrls: ['./elewa-vertical-icon-and-text.component.scss'],
 })
 export class ElewaVerticalIconAndTextComponent {
-  @Input() icon = "fa fa-user-group";
-  @Input() title = "Cooperative";
-  @Input() description = "Description goes here Description goes here Description goes here Description goes here Description goes here Description goes here Description goes here Description goes here";
+  properties=[
+    {
```

Comment:
why have you removed the inputs on this reusable component?
This is not part of your issue, your task is to reuse the component, not edit it unless the task requires it.